RELEASING.md

```diff
@@ -41,3 +41,4 @@ For Squares, membership is managed through the `Workflow Swift Owners` registry
     bundle exec pod trunk push WorkflowConcurrency.podspec --synchronous
     ```
 
+1. [Square specific] To make the new version available internally, update the [CocoaPods spec repo](go/cocoapods-specs).
```

Comment:
Nit: The go link points to the wrong place, still points to Bitbucket, should point to GH